packages/core/src/router.ts

```diff
@@ -108,7 +108,7 @@ export class Router<Context extends Omit<RestrictedCrawlingContext, 'enqueueLink
      * Registers default route handler.
      */
     addDefaultHandler<UserData extends Dictionary = GetUserDataFromRequest<Context['request']>>(
-        handler: (ctx: Omit<Context, 'request'> & { request: Request<UserData> }) => Awaitable<void>,
+        handler: (ctx: Omit<Context, 'request'> & { request: LoadedRequest<Request<UserData>> }) => Awaitable<void>,
```

Comment:
Isn't this just a redefinition of `LoadedContext`?

Reply:
nope, this also respects the user data type argument which you cant put into the `LoadedContext` type directly

Reply:
Gotcha, thanks